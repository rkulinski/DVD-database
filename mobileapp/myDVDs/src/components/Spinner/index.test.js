import React from 'react';
import { shallow } from 'enzyme';
import { Spinner } from './';


describe('Testing Header component', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <Spinner/>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
