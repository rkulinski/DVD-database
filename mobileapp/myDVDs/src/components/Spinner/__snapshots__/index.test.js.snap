// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Header component renders as expected 1`] = `
<Component>
  <ActivityIndicator
    animating={true}
    color="#999999"
    hidesWhenStopped={true}
    size="large"
  />
</Component>
`;
